graspy/embed/lse.py

```diff
@@ -71,6 +71,9 @@ class LaplacianSpectralEmbed(BaseEmbed):
         node degree is added. If int or float, must be >= 0. Only used when
         ``form`` == 'R-DAD'.
 
+    concat : bool, optional (default False)
+        if graph is directed whether to concatenate each graph's embedding along axis 1.
```

Comment:
in this description and all others, describe that this is concatenation of the "out" and "in" latent positions for the directed graph